-1

Tested at: 981df4bb7e91e2e76f01ba9af415bd83bfe0b6f9

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-16499/16232/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/DQMServices/Components/test/runTheWhiteRabbit.cpp 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/DQMServices/Components/test/DummyBookFillDQMStoreMultiThread.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/DQMServices/Components/test/DummyBookFillDQMStore.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/DQMServices/Components/test/DummyHarvestingClient.cc 
In file included from /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/EventFilter/CastorRawToDigi/plugins/CastorRawToDigi.cc:15:0:
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/CalibCalorimetry/HcalAlgos/interface/HcalDbASCIIIO.h:126:49: error: 'HcalCalibrationWidthsSet' does not name a type
   bool dumpObject (std::ostream& fOutput, const HcalCalibrationWidthsSet& fObject);
                                                 ^
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/DQMServices/Components/test/DummyTestReadDQMStore.cc 
>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/DQMServices/Components/src/DQMDcsInfo.cc 
>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-11-07-1100/src/DQMServices/Components/src/DQMStoreStats.cc 
</pre>

--- a/EventFilter/CastorRawToDigi/plugins/CastorRawToDigi.cc
+++ b/EventFilter/CastorRawToDigi/plugins/CastorRawToDigi.cc
@@ -12,7 +12,6 @@ using namespace std;
 #include <iostream>
 #include "EventFilter/CastorRawToDigi/interface/CastorRawCollections.h"
 #include "CalibCalorimetry/HcalPlugins/src/HcalTextCalibrations.h"
-#include "CalibCalorimetry/HcalAlgos/interface/HcalDbASCIIIO.h"
 #include "CondFormats/HcalObjects/interface/HcalElectronicsMap.h"
 #include "Geometry/CaloTopology/interface/HcalTopology.h"
 #include "FWCore/ParameterSet/interface/FileInPath.h"

